textblob/blob.py

```diff
@@ -148,6 +148,16 @@ def lemmatize(self, pos=None):
         lemmatizer = nltk.stem.WordNetLemmatizer()
         return lemmatizer.lemmatize(self.string, pos)
 
+    #added 'stemmer' on lines of lemmatizer
+    #based on nltk
+    def stem(self, isporter=True):
+        #param isporter: True when using Porter stemmer, else Snowball
```

Comment:
Please document this method with a docstring, using the Sphinx syntax (see other methods for examples).